CHANGES.md

```diff
@@ -1,3 +1,8 @@
+## Changes in 0.5.2 (in development)
+
+- Update of `pyproject.toml'file; package name changed and entry point removed, since xcube
+  plugins auto-recognition is updated. (#39 and xcube-dev/xcube#963)

```

Comment:
```suggestion
- Updated `pyproject.toml` file; package name changed from `xcube-cmems`
  to `xcube_cmems` and entry point removed, since xcube plugins 
  auto-recognition is updated. (#39 and xcube-dev/xcube#963)
```

Note, line length in `*.md` files should not exceed 80 chars.